Dockerfile

```diff
@@ -2,6 +2,9 @@
 FROM node:12.7-alpine AS build
 WORKDIR /usr/src/app
 COPY package.json package-lock.json ./
+RUN apk add --no-cache python2

```

Comment:
Do we need additional dependencies just because we change the registry?

Reply:
That is a fix to be able to build it on the arm64 architecture. I can't remember why it was necessary, because that commit is almost a year old.